packages/app-store/paypal/api/index.ts

```diff
@@ -1,3 +1,2 @@
 export { default as add } from "./add";
-export { default as webhook, config } from "./webhook";
```

Comment:
This is actually not a route we use as part of the catch all - its specifically handled in the main app/api/integrations route